metadata-ingestion/src/datahub/ingestion/source/s3/__init__.py

```diff
@@ -316,7 +317,7 @@ def read_file_spark(self, file: str, ext: str) -> Optional[DataFrame]:
             df = self.spark.read.json(file)
         elif ext.endswith(".avro"):
             try:
-                df = self.spark.read.format("avro").load(file)
+                df = self.sparkeread.format("avro").load(file)
```

Comment:
accidental edit ?

Reply:
Yeah this is still a work in progress. Not sure how this got in.

Reply:
No issues. Let us know when this is ready.

Reply:
Fixed this small change.